src/main/java/core/basesyntax/servicesimpl/DataProcessingServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.servicesimpl;
+
+import core.basesyntax.services.DataProcessingService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DataProcessingServiceImpl implements DataProcessingService {
+    public static final int FRUIT_INDEX = 1;
+    public static final int OPERATION_INDEX = 0;
+    public static final int QUANTITY_INDEX = 2;
```

Comment:
the same issue with access modifiers. Please also fix it in other classes

Reply:
+